source/01-EQ/02.ptx

```diff
@@ -418,14 +422,19 @@
         </statement>
         <answer>
           <p>
-            B.
+            B
           </p>
         </answer>
       </task>
 
       <task>
         <statement>
-          <p> We are almost ready to solve, but we have two variables in our weight equation and our cost equation. We will get the cost equation to one variable by using the weight equation as a substitution. Which of the following is a way to express one variable in terms of the other? (Hint: More than one answer may be correct here!)
+          <p> We are almost ready to solve, but we have two variables in our weight equation and our cost equation. We will get the cost equation to one variable by using the weight equation as a substitution. Which of the following is a way to express one variable in terms of the other?
+            <hint>
+              <p>
+                More than one answer may be correct here!
+              </p>
+            </hint>
```

Comment:
This isn't well-aligned with "Specific Choice". You could remove the alternative solution, warn students that they are just picking among the choices given, then add a remark following this activity outlining how the alternative solution could be used instead.

But this is out of scope for this PR's goals, so I'll open an issue.

Reply:
#435

Reply:
I think this hint should be a sibling to the statement, not a child?